internal/services/v1/schema.go

```diff
@@ -60,6 +62,10 @@ func (ss *schemaServer) ReadSchema(ctx context.Context, in *v1.ReadSchemaRequest
 		objectDefs = append(objectDefs, objectDef)
 	}
 
+	usagemetrics.SetInContext(ctx, &dispatchv1.ResponseMeta{
+		DispatchCount: uint32(len(nsDefs)),
```

Comment:
I wonder if this should be `+ 1`, since if their schema is empty, we are still doing *something*

Reply:
I didn't think so, it's easier to document that it's the number of schemas rather than `max(numSchemas, 1)`

Reply:
Like, if we fail the request, that's also counting as zero since we didn't really do any work on their behalf...

Reply:
Well, we are still making the request itself, but yeah